add-content.md

```diff
@@ -25,11 +36,11 @@ Prior to adding or editing content, all additions and changes must be approved a
 
 ### index.md
 
-Documentation will be created in or converted to the Markdown format (see [Using Markdown](/docs-guide/using-markdown)) to be leveraged by Jekyll to build the website. Add simple documentation like a short policy or one-page informational sheet to the GitHub repository as a single Markdown file (named `index.md`).
+`index.md` is a Markdown file that contains the first (or only) page of the documentation. When the site is built, it uses Jekyll to convert the Markdown to HTML (see [Using Markdown](/docs-guide/using-markdown)). Add simple documentation like a short policy or one-page workflow to the GitHub repository as a single Markdown file named `index.md`. For multi-page documentation, see the [Multiple Documentation Files](#multiple-documentation-files) section below.
```

Comment:
```suggestion
`index.md` is a Markdown file that contains the first (or only) page of the documentation. When the site is built, it uses Jekyll to convert the Markdown to HTML (see [Using Markdown](/docs-guide/using-markdown)). Simple documentation like a short policy or one-page workflow should be added to this file. For multi-page documentation, see the [Multiple Documentation Files](#multiple-documentation-files) section below.
```

Reply:
Feeling like the details of the Markdown format (and the mechanics of how the site is built) are not specific to this particular file, and somewhat distracting from what we're actually trying to communicate about this file.